src/containers/artist-dashboard-page/components/SendInputBody.tsx

```diff
@@ -0,0 +1,114 @@
+import { Button } from '@audius/stems'
+import BN from 'bn.js'
+import Input from 'components/data-entry/Input'
+import React, { useState } from 'react'
+import { WalletAddress } from 'store/wallet/slice'
+import { Nullable } from 'utils/typeUtils'
+import { ModalBodyWrapper } from '../WalletModal'
+
+// @ts-ignore
+window.bn = BN
```

Comment:
intentional?

Reply:
nope, this was from some testing. will remove